### PR TITLE
264 display data in verified user card

### PR DESCRIPTION
**Pull request by bogdan-evtushenko**

### CODE REVIEW

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/ProfileItemEpoxyModel.kt [hunk 1]
```diff
@@ -64,9 +72,77 @@ class ProfileItemEpoxyModel(
...
+    private fun User.buildMaxRating(context: Context) = SpannableString(
+            context.getString(
+                    R.string.max_rating_only,
+                    maxRating?.toString() ?: context.getString(R.string.none)
+            )
+    ).apply {
+        maxRating?.let {
+            val startIndex = indexOf(it.toString())
+            val color = getColorByUserRank(rank)
+            colorSubstring(startIndex, startIndex + it.toString().length, color)
```
Comment (MatyashDen) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/ProfileItemEpoxyModel.kt:144]:
```suggestion
        maxRating?.let {
            val startIndex = indexOf(it.toString())
            val color = getColorByUserRank(maxRank)
            colorSubstring(startIndex, startIndex + it.toString().length, color)
```

**Review by MatyashDen — APPROVED** [review]
Could you add border with color to user icon?

**Comment by MatyashDen** [comment]
Why after logout i am logged in again?

**Comment by bogdan-evtushenko** [comment]
> Why after logout i am logged in again?

Because we have to merge your PR in which one you clear the MyAccount in SharedPreferences. I don't do this logic for now.

File: app/src/main/res/layout/view_profile_item.xml [hunk 2]
```diff
@@ -23,6 +23,8 @@
             android:id="@+id/ivProfile"
             android:layout_width="48dp"
             android:layout_height="48dp"
+            app:civ_border_color="@color/colorPrimary"
```
Comment (MatyashDen) [inline app/src/main/res/layout/view_profile_item.xml:26]: What about color of user?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt [hunk 3]
```diff
@@ -46,6 +48,12 @@ class UserActivity : AppCompatActivity(), StoreSubscriber<UsersState> {
 
         setupChart()
 
+        menuItemId = if (handle == store.state.users.userAccount?.codeforcesUser?.handle) {
+            R.menu.menu_user_activity_log_out
+        } else {
+            R.menu.menu_user_activity_delete
+        }
+        // println("Here handle : ${handle}")
         store.dispatch(UsersRequests.FetchUser(handle))
```
Comment (MatyashDen) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt:57]: 👀

File: app/src/main/java/com/bogdan/codeforceswatcher/handlers/NotificationMiddleware.kt [hunk 4]
```diff
@@ -62,7 +61,7 @@ class AndroidNotificationHandler : NotificationHandler {
     @RequiresApi(android.os.Build.VERSION_CODES.O)
     private fun createNotificationChannel(
             context: Context,
-            notificationManager: NotificationManager
+            notificationManager: NotificationManager,
```
Comment (MatyashDen) [inline app/src/main/java/com/bogdan/codeforceswatcher/handlers/NotificationMiddleware.kt:64]: 🥇

**Review by MatyashDen — APPROVED** [review]
Could you add log out dialog when when press log out on user screen?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt [hunk 5]
```diff
@@ -30,12 +31,13 @@ import kotlinx.android.synthetic.main.activity_user.*
 import tw.geothings.rekotlin.StoreSubscriber
 import java.text.SimpleDateFormat
 import java.util.*
+import kotlin.properties.Delegates
 
 class UserActivity : AppCompatActivity(), StoreSubscriber<UsersState> {
 
-    private val handle
-        get() = intent.getStringExtra(HANDLE)
+    private val handle get() = intent.getStringExtra(HANDLE)
```
Comment (MatyashDen) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt:38]: Not sure about this code style

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt [hunk 6]
```diff
@@ -150,7 +163,7 @@ class UserActivity : AppCompatActivity(), StoreSubscriber<UsersState> {
 
         private const val HANDLE = "handle"
 
-        fun newIntent(context: Context, handle: String): Intent {
+        fun newIntent(context: Context, handle: String?): Intent {
             val intent = Intent(context, UserActivity::class.java)
```
Comment (MatyashDen) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt:167]: Why optional?

**Review by MatyashDen — APPROVED** [review]
Ship it!

**Comment by MatyashDen** [comment]
> Could you add log out dialog when when press log out on user screen?

Looks like you forgot about that

**Pull request merged**